python/ray/dashboard/modules/train/train_head.py

```diff
@@ -92,6 +94,44 @@ async def get_train_runs(self, req: Request) -> Response:
             content_type="application/json",
         )
 
+    async def _add_actor_status(self, train_runs):
+        try:
+            from ray.train._internal.state.schema import ActorStatusEnum, RunStatusEnum
+        except ImportError:
+            logger.exception(
+                "Train is not installed. Please run `pip install ray[train]` "
+                "when setting up Ray on your cluster."
+            )
```

Comment:
Do we need this in this method since it's already checked at the start of `get_train_runs`? Also if this ever does happen it'll just error out when trying to use these imports later in the code.

Reply:
Removed